keps/2724-topology-aware-scheduling/README.md

```diff
@@ -429,6 +429,14 @@ const (
   // at the highest topology level, then it gets admitted as distributed
   // among multiple topology domains.
   PodSetPreferredTopologyAnnotation = "kueue.x-k8s.io/podset-preferred-topology"
+
+  // PodSetRelaxedTopologyAnnotation indicates that a PodSet requires
+  // Topology Aware Scheduling, but it promotes filling up nodes to maximum over
```

Comment:
It is mixing up pod needs with cluster efficiency. Can we have these things separated. Annotation on a podset should mean that Kueue is free to place podset as it see fit in a given moment, and the current implementation is to fill the already utilozed nodes (but may change in the future).

Reply:
Done